server/controllers/entities/lib/move_to_wikidata.js

```diff
@@ -18,6 +18,7 @@ module.exports = (user, invEntityUri) => {
   })
   .then(entity => {
     const { labels, claims } = entity
+    rewriteP31(claims, asP31value)
```

Comment:
on the fly rewriting before pushing, allowed only because of entity destruction after pushing action, otherwise would require an entity update before pushing